modules/date-localizer/main/index.coffee

```diff
@@ -0,0 +1,168 @@
+class DateLocalizer
```

Comment:
I guess it does make sense - the reason I chose `DateLocalizer` is because it was shorter and technically it does only localize date objects - one of the formats happens to be time.

I'm fine with changing it to `DateTimeLocalizer`